pkg/api/mlflow/service/experiment/service.go

```diff
@@ -219,6 +221,16 @@ func DeleteExperiment(c *fiber.Ctx) error {
 		return api.NewInternalError("Unable to delete experiment '%d': %s", *exp.ID, tx.Error)
 	}
 
+	if tx := database.DB.Model(&run).Where("experiment_id = ?", exp.ID).Updates(&database.Run{
+		LifecycleStage: database.LifecycleStageDeleted,
+		DeletedTime: sql.NullInt64{
+			Int64: time.Now().UTC().UnixMilli(),
+			Valid: true,
```

Comment:
Probably this changes now a bit outdated, because all the services were changed. Can you please update this branch from main?